.github/workflows/ci.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
         image:
           - ubuntu-daily:jammy
           - ubuntu-daily:mantic
-          - ubuntu-daily:noble
+          # - ubuntu-daily:noble
```

Comment:
I don't usually like approving red CI builds, but with current noble archive state I'll OK it.  Let's leave this check in place please.

Reply:
Sounds good. Commit dropped!